### PR TITLE
[docs] Translate the FAQ(ZH) document to English and Replace the Conn…

### PR DESCRIPTION
**Pull request by rookiegao**
…ectors FAQ documents with the FAQ(ZH) document and FAQ link #725

### CODE REVIEW

**Comment by rookiegao** [comment]
# What is the purpose of the change
- Translate the FAQ documentation to English #725
- Merge the Connectors FAQ e.g. MySQL FAQ into the FAQ(ZH) document #725 
- Replace the Connectors FAQ document with the FAQ(ZH) document and FAQ link #725

**Comment by rookiegao** [comment]
@leonardBang The translated documents have been sent to your email. Please help me review and put forward better suggestions

File: docs/content/connectors/mongodb-cdc.md [hunk 1]
```diff
@@ -506,3 +506,9 @@ Reference
 - [Connection String Options](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-connection-options)
 - [BSON Types](https://docs.mongodb.com/manual/reference/bson-types/)
 - [Flink DataTypes](https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/table/types/)
+
+[FAQ](https://github.com/ververica/flink-cdc-connectors/wiki/FAQ)
+--------
+
+[FAQ(ZH)](https://github.com/ververica/flink-cdc-connectors/wiki/FAQ(ZH))
+--------
```
Comment (leonardBang) [inline docs/content/connectors/mongodb-cdc.md:514]: Thanks for the contribution @rookiegao . How bout changing the style to following?
```
FAQ
--------
* [FAQ(English)](https://github.com/ververica/flink-cdc-connectors/wiki/FAQ)
* [FAQ(中文)](https://github.com/ververica/flink-cdc-connectors/wiki/FAQ(ZH))
```

**Review by leonardBang — APPROVED** [review]
@rookiegao Thanks for the nice work, LGTM.
I‘ll merge it after update the WIKI page.

**Pull request merged**